cmd/internal/resolve/resolve.go

```diff
@@ -77,19 +77,28 @@ func AddTo(c commander.Interface) {
 
 			}
 
-			r := resolver.NewDefaultSatResolver(p, phonyCatalogSourceLister{}, log)
+			sp := resolver.SourceProviderFromRegistryClientProvider(p, logrus.New())
+
+			r := resolver.NewDefaultSatResolver(sp, phonyCatalogSourceLister{}, log)
 
 			var nsnames []string
 			for _, ns := range b.namespaces {
 				nsnames = append(nsnames, ns.GetName())
 			}
 
-			output, err := r.SolveOperators(nsnames, b.csvs, b.subscriptions)
+			operators, err := r.SolveOperators(nsnames, b.csvs, b.subscriptions)
 			if err != nil {
 				return fmt.Errorf("resolution failed: %w", err)
 			}
 
-			fmt.Fprintf(cmd.OutOrStdout(), "output: %v\n", output)
+			for _, operator := range operators {
+				fmt.Printf(
+					"---\nBundle: %s\nPath: %s\n",
+					operator.Name,
+					operator.BundlePath,
+				)
```

Comment:
This writes to `cmd.OutToStdout()` so that it plays nice when the default output writer is overridden here or in a parent command (via https://pkg.go.dev/github.com/spf13/cobra#Command.SetOut). Might as well keep using it unless it was causing a problem.

More importantly, `Name` may not be specific enough. This will output the same name regardless of the channel, package, catalog of origin, etc.